<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ToPaoPao</class>
 <widget class="QWidget" name="ToPaoPao">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>688</width>
    <height>592</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ToPaoPao</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" stretch="4,0">
   <item>
    <widget class="QScrollArea" name="scrollArea">
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">QScrollBar:vertical
{
    background: transparent;
    width: 6px;
    border: 0px;
    border-radius: 3px;
}
QScrollBar::handle:vertical
{
    background-color: #c3c0b9;
    border-radius: 3px;
    min-height:60px;
}

QScrollBar::handle:vertical:hover
{
    background-color: #686868;
    min-height:60px;
}

QScrollBar::handle:vertical:pressed
{
    background-color: #686868;
    min-height:60px;
}</string>
     </property>
     <property name="horizontalScrollBarPolicy">
      <enum>Qt::ScrollBarAlwaysOff</enum>
     </property>
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents_2">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>668</width>
        <height>416</height>
       </rect>
      </property>
      <property name="autoFillBackground">
       <bool>true</bool>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="widget" native="true">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>150</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>150</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">QWidget::widget {background-color: rgb(255, 255, 255);}</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2" stretch="2,0">
        <property name="spacing">
         <number>0</number>
        </property>
        <item>
         <widget class="QPlainTextEdit" name="texteditInput">
          <property name="styleSheet">
           <string notr="true">QPlainTextEdit {
border:none;
}</string>
          </property>
          <property name="plainText">
           <string/>
          </property>
          <property name="placeholderText">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QWidget" name="widget_2" native="true">
          <property name="styleSheet">
           <string notr="true">background-color: rgb(255, 255, 255);</string>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout_3">
           <property name="leftMargin">
            <number>0</number>
           </property>
           <property name="topMargin">
            <number>0</number>
           </property>
           <property name="rightMargin">
            <number>0</number>
           </property>
           <property name="bottomMargin">
            <number>0</number>
           </property>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout">
             <property name="rightMargin">
              <number>10</number>
             </property>
             <property name="bottomMargin">
              <number>3</number>
             </property>
             <item>
              <spacer name="horizontalSpacer">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
             <item>
              <widget class="QPushButton" name="btnSend">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="minimumSize">
                <size>
                 <width>75</width>
                 <height>25</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>75</width>
                 <height>25</height>
                </size>
               </property>
               <property name="styleSheet">
                <string notr="true">QPushButton{
background-color: rgb(245, 245, 245);
border:1px solid #E1E1E1;
}
QPushButton:hover{
background-color: rgb(18, 150, 17);
color:white
}
</string>
               </property>
               <property name="text">
                <string>发送</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
  <action name="actionenter">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/new/prefix1/me.jpg</normaloff>:/new/prefix1/me.jpg</iconset>
   </property>
   <property name="text">
    <string>enter</string>
   </property>
   <property name="shortcut">
    <string>Return</string>
   </property>
   <property name="shortcutVisibleInContextMenu">
    <bool>false</bool>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections/>
</ui>
